db/migrate/20240812102032_create_users_organizations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateUsersOrganizations < ActiveRecord::Migration[6.1]
```

Comment:
we might need a follow up migration to remove the organization_id from the user

Reply:
Yes, we do need one. We should create an issue for this before you merge/deploy this.

Reply:
#1977